package.json

```diff
@@ -38,6 +38,7 @@
   "dependencies": {
     "@babel/polyfill": "^7.4.4",
     "chardet": "^0.4.2",
+    "crypto-js": "^4.0.0",
```

Comment:
Do we need this module?

Reply:
@kmanaseryan I think yes because I try to use only `require('crypto)' but didn't work for a file from input to create a hash.
But if know how to do with only crypto, we can try to improve.

```
// call content function and generate MD5 hash from this content
  async hash() {
    const content = await this.contentArrayBuffer()
    const wordArray = CryptoJS.lib.WordArray.create(content);

    const hash = CryptoJS.MD5(wordArray).toString()
    return hash;
  }
```